<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns='http://www.w3.org/1999/xhtml' xml:lang='en' lang='en'>
<head>
<title>obexftp.git - obexftp</title>
<meta name='generator' content='cgit v0.10.1'/>
<meta name='robots' content='index, nofollow'/>
<link rel='stylesheet' type='text/css' href='/cgit-data/cgit.css'/>
<link rel='shortcut icon' href='/favicon.ico'/>
<link rel='alternate' title='Atom feed' href='http://pkgs.fedoraproject.org/cgit/obexftp.git/atom/obexftp-pkgconfig_requires.patch?h=master' type='application/atom+xml'/>
</head>
<body>
<div id='cgit'><table id='header'>
<tr>
<td class='logo' rowspan='2'><a href='/cgit/'><img src='/cgit-data/cgit.png' alt='cgit logo'/></a></td>
<td class='main'><a href='/cgit/'>index</a> : <a title='obexftp.git' href='/cgit/obexftp.git/'>obexftp.git</a></td><td class='form'><form method='get' action=''>
<select name='h' onchange='this.form.submit();'>
<option value='f10'>f10</option>
<option value='f11'>f11</option>
<option value='f12'>f12</option>
<option value='f13'>f13</option>
<option value='f14'>f14</option>
<option value='f15'>f15</option>
<option value='f16'>f16</option>
<option value='f17'>f17</option>
<option value='f18'>f18</option>
<option value='f19'>f19</option>
<option value='f20'>f20</option>
<option value='f21'>f21</option>
<option value='f7'>f7</option>
<option value='f8'>f8</option>
<option value='f9'>f9</option>
<option value='fc6'>fc6</option>
<option value='master' selected='selected'>master</option>
</select> <input type='submit' name='' value='switch'/></form></td></tr>
<tr><td class='sub'>obexftp</td><td class='sub right'>Jesse Keating</td></tr></table>
<table class='tabs'><tr><td>
<a href='/cgit/obexftp.git/'>summary</a><a href='/cgit/obexftp.git/refs/'>refs</a><a href='/cgit/obexftp.git/log/obexftp-pkgconfig_requires.patch'>log</a><a class='active' href='/cgit/obexftp.git/tree/obexftp-pkgconfig_requires.patch'>tree</a><a href='/cgit/obexftp.git/commit/obexftp-pkgconfig_requires.patch'>commit</a><a href='/cgit/obexftp.git/diff/obexftp-pkgconfig_requires.patch'>diff</a><a href='/cgit/obexftp.git/stats/obexftp-pkgconfig_requires.patch'>stats</a></td><td class='form'><form class='right' method='get' action='/cgit/obexftp.git/log/obexftp-pkgconfig_requires.patch'>
<select name='qt'>
<option value='grep'>log msg</option>
<option value='author'>author</option>
<option value='committer'>committer</option>
<option value='range'>range</option>
</select>
<input class='txt' type='text' size='10' name='q' value=''/>
<input type='submit' value='search'/>
</form>
</td></tr></table>
<div class='path'>path: <a href='/cgit/obexftp.git/tree/'>root</a>/<a href='/cgit/obexftp.git/tree/obexftp-pkgconfig_requires.patch'>obexftp-pkgconfig_requires.patch</a></div><div class='content'>blob: ded4a583330f3834fe99f0947f4a7ed99df6cced (<a href='/cgit/obexftp.git/plain/obexftp-pkgconfig_requires.patch'>plain</a>)
<table summary='blob content' class='blob'>
<tr><td class='linenumbers'><pre><a id='n1' href='#n1'>1</a>
<a id='n2' href='#n2'>2</a>
<a id='n3' href='#n3'>3</a>
<a id='n4' href='#n4'>4</a>
<a id='n5' href='#n5'>5</a>
<a id='n6' href='#n6'>6</a>
<a id='n7' href='#n7'>7</a>
<a id='n8' href='#n8'>8</a>
<a id='n9' href='#n9'>9</a>
<a id='n10' href='#n10'>10</a>
<a id='n11' href='#n11'>11</a>
</pre></td>
<td class='lines'><pre><code>--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -71,7 +71,7 @@
 set ( exec_prefix "\${prefix}" )
 set ( libdir      "\${prefix}/${CMAKE_INSTALL_LIBDIR}" )
 set ( includedir  "\${prefix}/${CMAKE_INSTALL_INCLUDEDIR}" )
-set ( REQUIRES    "bluetooth openobex" )
+set ( REQUIRES    "bluez openobex" )
 configure_file (
   ${CMAKE_CURRENT_SOURCE_DIR}/obexftp.pc.in
   ${CMAKE_CURRENT_BINARY_DIR}/obexftp.pc
</code></pre></td></tr></table>
</div> <!-- class=content -->
<div class='footer'>generated  by cgit v0.10.1 at 2014-12-12 10:03:43 (GMT)</div>
</div> <!-- id=cgit -->
</body>
</html>
